jquery.instagramFeed.js

```diff
@@ -180,8 +181,25 @@
                         var imgs = (data.edge_owner_to_timeline_media || data.edge_hashtag_to_media).edges;
                         max = (imgs.length > options.items) ? options.items : imgs.length;
 
+                        if (options.random) {
+                            for (var i = 0; i < max; i++) {	                        var total = (12 < max) ? max : 12;
```

Comment:
What is `var total = (12 < max) ? max : 12;` for? There might be more than 12 elements for tags and, for users it is not a limitation we add, it comes from instagram.